src/Faker/Generator.php

```diff
@@ -557,6 +557,10 @@
  * @property string $uuid
  *
  * @method string uuid()
+ *
+ * @property string $semver
```

Comment:
@Nyholm @bramceulemans are we going to expose this for a new implementation too?

I do understand we do this for all the legacy loaders so we can get rid of all the old stuff but there are no current implementations on this one.

What is your view? I think we can skip this though

Reply:
Ah my mistake. For the blood type extension there are also no phpdocs and my method related docs are useless anyway, since the `semver` method exists in this class. Will remove it.

Reply:
Ok great, it is more there for legacy since that is all loaded dynamically. For the new things we want a proper strict implementation :)